app/services/combined_audio_derivative_creator.rb

```diff
@@ -0,0 +1,170 @@
+require 'tempfile'
+
+
+Response = Struct.new(:webm_file, :mp3_file, :fingerprint, :start_times, keyword_init: true)
```

Comment:
Nest this inside of `CombinedAudioDerivativeCreator`, by putting it under the `class CombinedAudioDerivativeCreator`. 

As you've done it, it's a top-level constant `Response`, which could conflict with something else trying to make a top-level `Response`. Instead, it'll be `CombinedAudioDerivativeCreator::Response`, avoiding name conflicts and making it clear where it comes from.

Reply:
Good call!